kiwi/term.h

```diff
@@ -18,14 +18,18 @@ class Term
 
 public:
 
-	Term( const Variable& variable, double coefficient = 1.0 ) :
-		m_variable( variable ), m_coefficient( coefficient ) {}
+	Term( Variable variable, double coefficient = 1.0 ) :
```

Comment:
Isn't this introducing a backward incompatibility ?

Reply:
It does not, passing parameters by value or const ref is the same on the caller side. This just allows for movement